### DISCUSSION TITLE
How to Auto run when raspi startup?

### DISCUSSION THREAD

**Discussion by aanarewak** [scivision/PyLivestream#89] [Q&A]
How to Auto run when raspi startup?
Please Help Me!!!

**Comment by the-iconic-rihan** [comment]
> How to Auto run when raspi startup? Please Help Me!!!



> How to Auto run when raspi startup? Please Help Me!!!

Hey I'm also have the same question. I also want the solution for the same

**Comment by scivision** [comment]
In general on a system with cron, most programs / scripts can be [run at startup via cron](https://www.linode.com/docs/guides/run-jobs-or-scripts-using-crontab-on-boot/#use-crontab-to-schedule-a-job-or-script-to-run-at-system-startup)
